word_guess.rb

```diff
@@ -0,0 +1,163 @@
+#word guess game
+
+#
+require "colorize"
+
+class Word_guess
+	attr_accessor :wrong_count, :word_list, :secret_word, :game_word, :game_word_array
```

Comment:
`attr_accessor` can be pretty dangerous. Are you sure you want me to be able to access `secret_word` when I'm playing your game? Or to be able to change `secret_word` from outside the class? You only need getter/setter methods when things (methods, other classes, users) need access to instance variables from _outside the class_. What would happen if you deleted this line completely? Would anything break?

`@word_list`, for example, is used three times in your program, none of which involve exposing the value to something/someone outside the object. Something to keep in mind in the future.